src/content/docs/en/reference/api-reference.mdx

```diff
@@ -52,14 +52,19 @@ Markdown files have the following interface:
 export interface MarkdownInstance<T extends Record<string, any>> {
```

Comment:
I’m wondering if we should mention `MarkdownInstance` in the introductory line above. Currently there are no search results for it because code samples are excluded from indexing.

For example something like

> Markdown files loaded with `Astro.glob()` return the following `MarkdownInstance` interface: